app/components/ProtectedRoute/index.js

```diff
@@ -43,7 +40,7 @@ const ProtectedRoute = ({ isUserVerified, render: C, isLoggedIn, handleLogout, .
         return <C {...renderProps} />;
       }
     }
```

Comment:
isLoggedIn is a boolean value, so if else instead of else if

Reply:
Updated